src/test/java/teammates/test/cases/ui/browsertests/InstructorCourseStudentDetailsEditPageUiTest.java

```diff
@@ -85,9 +85,15 @@ public void testInputValidation() {
             .verifyStatus(String.format(FieldValidator.SIZE_CAPPED_NON_EMPTY_STRING_ERROR_MESSAGE, 
                     "", teamNameFieldName, FieldValidator.REASON_EMPTY, teamNameFieldName, FieldValidator.TEAM_NAME_MAX_LENGTH));
         
-        String invalidStudentName = StringHelper.generateStringOfLength(FieldValidator.PERSON_NAME_MAX_LENGTH + 1);
+        ______TS("the team field is edited");
         String newTeamName = "New teamname";
-        editPage.submitUnsuccessfully(invalidStudentName, newTeamName, null, null)
+        editPage.submitUnsuccessfully("", newTeamName, null, null)
+            .verifyStatus(String.format(FieldValidator.SIZE_CAPPED_NON_EMPTY_STRING_ERROR_MESSAGE,
+                    "", personNameFieldName, FieldValidator.REASON_EMPTY, personNameFieldName, FieldValidator.PERSON_NAME_MAX_LENGTH));
+        
+        ______TS("the team field is not edited");
```

Comment:
Although this segment makes sense in this PR, in future developers may be confused on why this segment is labeled as such. This test is actually testing that a long student name is rejected, and should be labeled so.